packages/mcp-client/cli.ts

```diff
@@ -11,6 +11,7 @@ import { version as packageVersion } from "./package.json";
 
 const MODEL_ID = process.env.MODEL_ID ?? "Qwen/Qwen2.5-72B-Instruct";
 const PROVIDER = (process.env.PROVIDER as InferenceProvider) ?? "nebius";
+const ENDPOINT_URL = process.env.ENDPOINT_URL;
```

Comment:
```suggestion
const BASE_URL = process.env.BASE_URL;
```

don't you want to name it `BASE_URL`?

(feels weird to have `baseUrl: ENDPOINT_URL`  afterwards + `baseUrl` seems more future-proof given internal convo)

Reply:
yes, merging